CHANGES.rst

```diff
@@ -110,3 +113,4 @@ Released 2017-04-14
 .. _@minrk: https://github.com/minrk
 .. _@nuest: https://github.com/nuest
 .. _@yuvipanda: https://github.com/yuvipanda
+.. _@jrbourbeau: https://github.com/jrbourbeau
```

Comment:
Tried to sort user names alphabetically to make it easy to check if someone is already in the list, can you move yourself up?